lib/routes/admin/index.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+module.exports = function(mediator) {
+  const router = express.Router();
+
+  if (process.env.NODE_ENV === 'development') {
+    /**
+     * Asks for data to be reset to the default list of users
```

Comment:
Might be good to add caps here.. just to make doubly sure that developers know it should be removed.